internal/zero/cmd/command_import.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pomerium/pomerium/config"
+	"github.com/pomerium/pomerium/internal/log"
+	"github.com/pomerium/pomerium/pkg/envoy/files"
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+)
+
+func BuildImportCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import",
+		Short: "Import an existing configuration to a Zero cluster",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			configFlag := cmd.InheritedFlags().Lookup("config")
+			var configFile string
+			if configFlag != nil {
+				configFile = configFlag.Value.String()
+			}
+			if configFile == "" {
```

Comment:
There are three possible command templates that zero gives you to copy/paste depending on your environment:

linux: `pomerium zero import --token <token> --config /path/to/config.yaml` this uses an explicit --config flag.

kubernetes: `kubectl exec -it pomerium -- /bin/pomerium zero import --token <token>` here, the config file could be in a number of different places, so to avoid the user needing to look up where it is, we can just check where the currently running pomerium instance is reading it from. 

docker: `docker exec -it pomerium /bin/pomerium zero import --token <token>` same thing here, though it is probably more likely that it's being read from /pomerium/config.yaml since that is what we have in the docs. 

If we can reliably detect where the config file is, that will be less error-prone and fewer steps for the user.